modules/core-updates/9.0.md

```diff
@@ -85,6 +85,40 @@ Some dependencies are still present but were upgraded which comes with their own
 - [Services translator, security.csrf.token_manager, session, session.flash_bag no longer public](https://github.com/PrestaShop/PrestaShop/pull/35069)
 - [Replace AdminSecurity annotation with attributes](https://github.com/PrestaShop/PrestaShop/pull/35212)
 
+### BO login and authorization migrated to Symfony
+
+The login page to the back-office has been migrated to Symfony, along ith it the system of authorization in the BackOffice is now also based on Symfony, it implies several things:
+- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side and very few data are kept on the browser side, for retro-compatibility we still populate the legacy cookie so that you can **read** it but modifying/writing it will likely no longer have the effect you would expect (if any)
+- the `PrestaShopBundle\Security\Admin\EmployeeProvider` and `PrestaShopBundle\Service\DataProvider\UserProvider` now return a `PrestaShopBundle\Entity\Employee` instance, their responsibility as to return a `Symfony\Component\Security\Core\User\UserInterface` anyway (and they still do) but in case you depended on the child class know that `PrestaShopBundle\Security\Admin\Employee` no longer exist
```

Comment:
```suggestion
- the `PrestaShopBundle\Security\Admin\EmployeeProvider` and `PrestaShopBundle\Service\DataProvider\UserProvider` now return a `PrestaShopBundle\Entity\Employee` instance, their responsibility is to return a `Symfony\Component\Security\Core\User\UserInterface` anyway (and they still do) but in case you depended on the child class know that `PrestaShopBundle\Security\Admin\Employee` no longer exist
```